Update variable names and language used

mcdonnnj opened the pull request:
# <!-- Use the title to describe PR changes in the imperative mood --> #

## 🗣 Description ##

This pull request renames some variables, adjusts their descriptions, renames some facts, and adjusts some task names.
<!-- Describe the "what" of your changes in detail. -->
<!-- To avoid scope creep, limit changes to a single goal. -->

## 💭 Motivation and context ##

While working on #48 I noticed some inconsistencies and/or not quite accurate information for some variables, facts, and task names. This pull request simply seeks to clean that up and better document the functionality in this role.
<!-- Why is this change required? -->
<!-- What problem does this change solve? How did you solve it? -->
<!-- Mention any related issue(s) here using appropriate keywords such -->
<!-- as "closes" or "resolves" to auto-close them on merge. -->

## 🧪 Testing ##

Automated tests pass.
<!-- How did you test your changes? How could someone else test this PR? -->
<!-- Include details of your testing environment, and the tests you ran to -->
<!-- see how your change affects other areas of the code, etc. -->

<!--
## 📷 Screenshots (if appropriate) ##

Uncomment this section if a screenshot is needed.

-->

## ✅ Pre-approval checklist ##

<!-- Remove any of the following that do not apply. -->
<!-- Draft PRs should have one or more unchecked boxes. -->
<!-- If you're unsure about any of these, don't hesitate to ask. -->
<!-- We're here to help! -->

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - *eschew scope creep!*
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

mcdonnnj committed:
Update variable names for storing the database file

The updated variable names and descriptions better represent what these
variables store. A fact name is updated to reflect these changes to the
variable names.

mcdonnnj committed:
Update some task names

Better describe what's being done and remove a legacy reference to md5.

mcdonnnj committed:
Update the name of a registered value

Since the type of archive downloaded can be changed it makes sense to use
a more general name for this registered value.

mcdonnnj committed:
Change the names of two facts

There's no real reason to mention that the URLs are remote so we can
simplify the fact names. The task names for setting these values are
similarly updated.

jsf9k commented on README.md:
```diff
@@ -63,8 +63,8 @@ None.
-| geoip2\_local\_file | The filename used to store the downloaded database. | `GeoIP2-City.tar.gz` | No |
-| geoip2\_local\_path | The directory to extract the database into. | `/usr/local/share/GeoIP/` | No |
+| geoip2\_archive\_filename | The filename to use for the downloaded database archive. | `GeoIP2-City.tar.gz` | No |
+| geoip2\_database\_directory | The directory to store the database files. | `/usr/local/share/GeoIP/` | No |
```
```suggestion
| geoip2\_database\_directory | The directory in which to store the database files. | `/usr/local/share/GeoIP/` | No |
```

jsf9k commented on defaults/main.yml:
```diff
@@ -1,7 +1,9 @@
+# Name to use for the downloaded database archive
+geoip2_archive_filename: GeoIP2-City.tar.gz
+
 # Directory to store GeoIP files
```
```suggestion
# Directory in which to store GeoIP files
```

jsf9k commented on tasks/main.yml:
```diff
@@ -45,47 +45,47 @@
       }}
 
-- name: Create the local directory
+- name: Ensure the directory to store the database file exists
```
```suggestion
- name: Ensure the directory in which to store the database file exists
```

jsf9k reviewed (approved):
I had a few suggestions to increase readability.

mcdonnnj committed:
Improve language in some comments and descriptions

Better phrase references to storing MaxMind database files in a target
directory.

Co-authored-by: Shane Frasier <jeremy.frasier@gwe.cisa.dhs.gov>
